docker-compose.yml

```diff
@@ -62,20 +53,21 @@ services:
       - SPRING_PROFILES_ACTIVE=dev
 
   localstack:
-    image: localstack/localstack:0.11.2
+    image: localstack/localstack:0.12.10
     networks:
       - hmpps
     container_name: localstack
     ports:
       - "4566-4597:4566-4597"
       - 8999:8080
     environment:
-      - SERVICES=sqs,sns,es
+      - SERVICES=sqs,sns,es,elasticsearch
       - ES_PORT_EXTERNAL=4571
       - DEBUG=${DEBUG- }
       - DATA_DIR=/tmp/localstack/data
       - DOCKER_HOST=unix:///var/run/docker.sock
       - DEFAULT_REGION=eu-west-2
+      - HOSTNAME_EXTERNAL=localstack
```

Comment:
tricky one this.  When running prisoner offender search in docker this need to be `localstack` so that it can connect, however when running in intellij it need to be `localhost` instead.  I've therefore created a `docker-compose-localstack-tests.yml` file for when running the tests.